plugins/woocommerce-blocks/assets/js/icons/library/filter-three-lines.tsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import { SVG } from '@wordpress/primitives';
+
+const filterThreeLines = (
+	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+		<path
+			d="M4 11.5H8V10H4V11.5ZM0 0V1.5H12V0H0ZM2 6.5H10V5H2V6.5Z"
+			transform="translate(6,6)"
+		/>
+	</SVG>
```

Comment:
Should we import `Path` from the `@wordpress/primitives` library?

```suggestion
import { SVG, Path } from '@wordpress/primitives';

const filterThreeLines = (
	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
		<Path
			d="M4 11.5H8V10H4V11.5ZM0 0V1.5H12V0H0ZM2 6.5H10V5H2V6.5Z"
			transform="translate(6,6)"
		/>
	</SVG>
```